lib/options/dcu.robot

```diff
@@ -48,3 +48,29 @@ Get UEFI Option
     ...    cd dcu && ./dcu v coreboot.rom --get "${option_name}"
     ...    shell=True
     RETURN    ${out.stdout}
+
+Execute Warmboot Command
```

Comment:
Please move these out of `lib/options/dcu.robot` . I think `lib/sleep-lib.robot` fits these better

Reply:
I wrote those in `lib/options/dcu.robot` because I assumed the test suite may be also used to test devices via serial port. In that case any differences in performing those operations could be addressed by redefining the keywords in `lib/options/uefi-setup-menu.robot` .
Nevertheless its moved to `lib/sleep-lib.robot` now.